src/project.js

```diff
@@ -57,7 +63,22 @@ function project(location) {
   if (typeof target === 'string') {
     newLocation.target = target;
   }
-  return newLocation;
+
+  const heightLevelList = height;
+  initIterValue += 1;
+  let result = newLocation;
```

Comment:
Do we need result as a intermediate variable. Can't we use newLocation directly?

Reply:
Much better.